cilium/CFP-33716-packet-tracing.md

```diff
@@ -0,0 +1,636 @@
+# CFP-2024-06-12: Packet Tracing
+
+## Meta
+**SIG:** SIG-Hubble, SIG-Datapath  
+**Sharing:** Public  
+**Begin Design Discussion:** Jun 12, 2024  
+**End Design Discussion:**  
+**Cilium Release:**  
```

Comment:
I would guess the target is during the next six months, so we can put that in.

```suggestion
**End Design Discussion:**  Aug 16, 2024
**Cilium Release:**  1.17
```

Reply:
Fixed and pushed new changes.